### PR TITLE
feat: close folders for free users

### PR DESCRIPTION
**Pull request by ilasw**
Close feature for free users, now only Plus users should have the ability to create/edit/[...] a Bookmark Folder.

### CODE REVIEW

File: src/schema/compatibility.ts [hunk 1]
```diff
@@ -216,15 +215,13 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
...
     bookmarks: async (source, args, context: AuthContext, info) => {
-      const firstFolderId = await getFirstFolderId(context);
       const resolver = compatFeedResolver((ctx, args, opts, builder, alias) =>
         bookmarksFeedBuilder({
           ctx,
           unreadOnly: false,
           reminderOnly: false,
           builder: builder.orderBy('bookmark.createdAt', 'DESC'),
           alias,
-          firstFolderId,
```
Comment (sshanzel) [inline src/schema/compatibility.ts:227]: Let's also remove this prop from the interface itself.

**Review by sshanzel — APPROVED** [review]
Looks amazing to me! Kindly wait for Ante's review before merging 🙏

File: src/schema/compatibility.ts (hunk 1 above)
Comment (ilasw) [reply src/schema/compatibility.ts:227]: I think I've removed it, didn't I? 👀

File: src/schema/bookmarks.ts [hunk 2]
```diff
@@ -382,6 +385,12 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       { id, listId }: { id: string; listId?: string },
       ctx: AuthContext,
     ): Promise<GQLEmptyResponse> => {
+      if (!ctx.isPlus) {
+        throw new ForbiddenError(
+          'You need to be a Plus member to use this feature',
```
Comment (capJavert) [inline src/schema/bookmarks.ts:390]: let's export this to errors with other error labels

**Review by capJavert — APPROVED** [review]

**Review by omBratteng — APPROVED** [review]

File: src/schema/bookmarks.ts (hunk 2 above)
Comment (ilasw) [reply src/schema/bookmarks.ts:390]: Update ✔️ 🙏

**Review by rebelchris — APPROVED** [review]

**Pull request merged**